.github/bot_config.yml

```diff
@@ -67,7 +67,9 @@ cuda_comment: >
    
    | :-------------: | :-------------: |
    
-   | 2.1.0 - 2.2.0  | 10.1 |
+   | 2.4.0  | 11.0 |
```

Comment:
| TensorFlow pip packages are now built with CUDA11.2

Can you add 11.2 for 2.5.0 now that we have a pip RC?

Reply:
done